packages/block-library/src/rss/index.php

```diff
@@ -33,9 +33,9 @@ function render_block_core_rss( $attributes ) {
 		$link = $item->get_link();
 		$link = esc_url( $link );
```

Comment:
I think we don't need this here anymore.

Reply:
Good spot. Fixed!